EuchreSolution/Euchre/GameScript/trick.ts

```diff
@@ -91,10 +91,8 @@ class Trick {
 		animShowText(Player[this.__currentPlayer] + " played " + getCardShorthand(card), MessageLevel.Step, 1);
```

Comment:
Name that correctly

Reply:
Merged with previous commit (like I meant to originally)